core/domain/playthrough_issue_registry_test.py

```diff
@@ -18,14 +18,57 @@
 
 from __future__ import annotations
 
+from core import utils
 from core.domain import playthrough_issue_registry
 from core.tests import test_utils
+from extensions.issues.CyclicStateTransitions import CyclicStateTransitions
+from extensions.issues.EarlyQuit import EarlyQuit
+from extensions.issues.MultipleIncorrectSubmissions import (
+    MultipleIncorrectSubmissions)
 
 
 class IssueRegistryUnitTests(test_utils.GenericTestBase):
     """Test for the issue registry."""
 
+    def setUp(self):
+        super(IssueRegistryUnitTests, self).setUp()
+        self.issues_dict = {
+            'EarlyQuit': (
+                EarlyQuit.EarlyQuit),
+            'CyclicStateTransitions': (
+                CyclicStateTransitions.CyclicStateTransitions),
+            'MultipleIncorrectSubmissions': (
+                MultipleIncorrectSubmissions.MultipleIncorrectSubmissions)
+            }
+        self.invalid_issue_type = 'InvalidIssueType'
+
+    def tearDown(self):
+        super(IssueRegistryUnitTests, self).tearDown()
+        playthrough_issue_registry.Registry._issues = {} # pylint: disable=protected-access
```

Comment:
Probably cleaner conceptually to do this first before calling super().

Reply:
yupp !!